lms/validation/_api.py

```diff
@@ -12,15 +12,6 @@ class APIRecordSpeedgraderSchema(JSONPyramidRequestSchema):
     document_url = fields.Str()
```

Comment:
This should be made required=True.

While we deploy this I rather see an exception from the `parsed_documents['document_url']` below instead of emitting a HTTP 400 that would be missed in the logs.